portfolio/src/main/webapp/procrastination.html

```diff
@@ -11,6 +11,7 @@
         <h1>sehhaj's portfolio - still to be implementated</h1>
     </div>
   </body>
-
+  
+  <div id="nav-bar">
```

Comment:
Generally speaking it's bad practice to slip in a change for an unrelated project, this should really have been a seperate PR (but I'll pretend I didn't see it this time :P )

Reply:
Yes I will keep a look out for this in the future. I think I forgot to push the commit last week so it was added in today commit?